config/config.go

```diff
@@ -173,18 +173,34 @@ func NewConfig(path string) (*Config, error) {
 	c := newConfig()
 	c.ConfigFile = path
 
+	log.Debugf("Loading choria configuration from '%s'", path)
```

Comment:
I'd remove these 2 log lines, useful when developing but this early on logging is not configured yet and we might end up spewing this to peoples console/stderr at times, it's probably fine but I just try to avoid logging here entirely except for warning/errors

Reply:
Yeah, I was concerned that the config can change while parsing the configuration files, but was happy to see that passing `-d` had the expected behavior…  But this piece of information is not very useful, so I'll remove it.